gapic-generator/lib/gapic/presenters/snippet_presenter.rb

```diff
@@ -118,8 +118,11 @@ def snippet_name
 
       def description
         @config&.metadata&.snippet_description ||
-          "This is an auto-generated example demonstrating basic usage of #{client_type}##{method_name}. " \
-          "It may require modification in order to execute successfully."
+          "This snippet has been automatically generated and should be regarded as a code template only." \
```

Comment:
The backslashes will only concatenate strings directly. You'll need to explicitly embed newlines (`\n`) in the string if you want them formatted on separate lines.

Reply:
Thanks Daniel! The codegen wraps comments to a specific length & trims whitespace (both ends), so I added the non-ASCII space in order to keep the docs align (i.e. \u00A0). PLMK if you have a better method.

The rest done via`toys bin` + `toys gen`.

Reply:
The double `\n` approach did not work, but I updated the template file accordingly. It renders correct in HTML -- I will send you the image in chat.